src/test/java/com/powsybl/openloadflow/ac/DistributedSlackOnGenerationTest.java

```diff
@@ -60,6 +60,8 @@ void setUp() {
     void test() {
```

Comment:
I have tried this use case with a distribution on loads and it seems that we have a or several sign issues...